### DISCUSSION TITLE
NOTE TAKING EXTENSION

### DISCUSSION THREAD

**Discussion by knana1662** [microsoft/vscode-discussions#540] [Extension Development QnA] [answered]
Please I would like to ask if someone could help me find the name of the extension below and also provide a link to it. Thank You
<img width="286" alt="primary-sidebar1" src="https://user-images.githubusercontent.com/109052744/227724615-0ffd3d11-6681-433e-a7e7-bb9eb6608fa8.png">

**Comment by isidorn** [comment]
Not sure. But you could google this. Or maybe event put this image into ChatGPT with internet access and it should be able to answer.

**Comment by burkeholland** [comment, marked as answer]
I see you found this in our UX guidelines. I don't think that this extension actually exists. I believe this is a mockup to demonstrate the UX guidelines. That said - it's a free idea! Go for it! https://code.visualstudio.com/api/ux-guidelines/views

**Comment by gziz** [comment]
Hi @knana1662 . A bit late to the party. Yet, I also wanted/needed an extension similar to the image one. Proceeded to create one myself. Hope it can help. 
[SnipNotes MarketPlace URL](https://marketplace.visualstudio.com/items?itemName=gziz.snip-notes)
